documentation/modules/ROOT/pages/modules/tutorial/p_viewing-create-event.adoc

```diff
@@ -334,7 +334,7 @@ However, the Kafka topic containing all of the events for a single table might h
 
 The JSON converter includes the key and value schemas in every message,
 so it does produce very verbose events.
-Alternatively, you can use the link:http://docs.confluent.io/3.1.2/schema-registry/docs/index.html[Avro converter], which results in far smaller event messages.
+Alternatively, you can use xref:configuration/avro.adoc[Apache Avro] as a serialization format, which results in far smaller event messages.
```

Comment:
I believe we probably should add the cdc-product conditional here as any other prior uses in the downstream docs that reference `configuration/avro.adoc` are wrapped using `ifndef::cdc-product[]`.

Reply:
Added.